mcs/class/System.Private.CoreLib/System/Exception.cs

```diff
@@ -4,8 +4,6 @@ namespace System
 {
 	partial class Exception
 	{
-		protected event EventHandler<SafeSerializationEventArgs> SerializeObjectState;
```

Comment:
This API is required

Reply:
This adds fields to the Exception class whose layout the runtime depends on.

Reply:
Then we need to fix it differently, the explicit event probably won't help as the partial order is not guaranteed